libraries/simple.js.yaml

```diff
@@ -0,0 +1,13 @@
+name: simple.js
+description: Helper functions and defaults for young and new coders.
+category: learning
```

Comment:
Hi! Currently, we're trying to fit libraries into one of our existing categories. Is there a category from the readme that best fits this project?

Reply:
@davepagurek sadly none of the existing categories fits this project.  I think there is a case for a new category "teaching".

"accessibility" if that already exists.

Reply:
I think we can make something work! There are some other libraries that can maybe fit into a teaching category, so I'll turn it into that for now.